docs/test-run.md

```diff
@@ -0,0 +1,114 @@
+
+The Certsuite binary can be built as follows:
+
+```shell
+make build-certsuite-tool
+```
+
+## Test labels
+
+The test cases cases have several labels to allow for different types of groupings when selecting which to run. These are the following:
+
+* The name of the test case
+* The name of the test suite
+* The category of the test case (common, telco, faredge, extended)
+
+These labels can be combined with some operators to create label filters that match any condition. For example:
+
+* The label filter "observability,access-control" will match the test suites _observability_ and _access-control_.
+* The label filter "operator && !operator-crd-versioning" will match the _operator_ test suite without the _operator_crd_versioning_ test case.
+* To select all the test cases the _all_ label filter can be used.
+
+To view which test cases will run for a specific label or label filter use the flag `--list`.
+
+See the [CATALOG.md](CATALOG.md) to find all test labels.
+
+## Selected flags description
+
+The following is a non-exhaustive list of the most common flags that the `certsuite run` command accepts. To see the complete list use the `-h, --help` flag.
+
+* `-l, --label-filter`: Label expression to filter test cases. Can be a test suite or list or test suites, such as `"observability,access-control"` or a more complex expression with logical operators such as `"access-control && !access-control-sys-admin-capability"`.
+
+!!! note
+
+    If `-l` is not specified, the Test Suite will run in 'diagnostic' mode. In this mode, no test case will run: it will only get information from the cluster (PUTs, CRDs, nodes info, etc…) to save it in the claim file. This can be used to make sure the configuration was properly set and the autodiscovery found the right pods/crds…
+
+* `-o, --output-dir`: Path of the local directory where test results (claim.json), the execution logs (certsuite.log), and the results artifacts file (results.tar.gz) will be available from after the container exits.
+
+* `-k, --kubeconfig`: Path to the Kubeconfig file of the target cluster.
+
+* `-c, --config-file`: Path to the `tnf_config.yml` file.
+
+* `--preflight-dockerconfig`: Path to the Dockerconfig file to be used by the Preflight test suite
+
+* `--offline-db`: Path to an offline DB to check the certification status of container images, operators and helm charts. Defaults to the DB included in the test container image.
+
+!!! note
+
+    See the [OCT tool](https://github.com/test-network-function/oct) for more information on how to create this DB.
+
+## Using the container image
+
+The only prerequisite for running the Test Suite in container mode is having Docker or Podman installed.
+
+### Pull the test image
+
+The test image is available at this repository [repository](https://quay.io/repository/testnetworkfunction/cnf-certification-test) and can be pulled using:
+
+```shell
+docker pull quay.io/testnetworkfunction/cnf-certification-test
```

Comment:
Should we specify `:latest` here?  Or add some text about pulling a desired release version?